### DISCUSSION TITLE
Testing changes

### DISCUSSION THREAD

**Discussion by ShuraxN** [OpenLoco/OpenLoco#2460] [Q&A] [answered]
Hey guys,
I am pretty new to Open-Source-Projects and wanted to test how some changes on the source code create changes on the game. So I have tried to change the daycount for the date but when I compile it the date stays the same. Does anyone know what I did wrong or whats the fastest method to see changes ingame so I can confirm that I am able to mod this project.

**Comment by AaronVanGeffen** [comment]
You'll need to recompile the source code before any changes will be visible in game. Could you tell us something about your setup? What platform are you using, what compiler, and any other potentially useful info?

If you'd like us to help you with some code, the easiest way is probably to fork the repository and push your changes to a branch on your fork. You can then either link the branch here, or make a pull request if you think it'll be useful to a wider audience. (In this case, changing the date logic is not something we're likely to accept at this time, though.)

**Reply by ShuraxN** [reply]
I am currently using a Linux system. The changing of the date was just a test to see if I change something on the source code, it would be visible in the game. For example I went into the Date.cpp file and tried to change the pair id  { MonthId::january, 1 } to  { MonthId::january, 2 } and compiled the game again with cmake --build --preset linux-release. But the change would not be visible ingame for me.

**Reply by duncanspumpkin** [reply, marked as answer]
If you want to see something change it would be easier to just change something in a Window function. For example https://github.com/OpenLoco/OpenLoco/blob/3188a084432b3af808ede9898e488c7647aac195/src/OpenLoco/src/Windows/Cheats.cpp#L428 if you changed this line to a fixed value you would see that changes the cheats window currency step.
